libs/ledgerjs/packages/types-live/src/account.ts

```diff
@@ -123,7 +123,7 @@ export type Account = {
   // A sub account can be either a token account or a child account in some blockchain.
   // They are attached to the parent account in the related blockchain.
   // CONVENTION:
-  // a SubAccount is living inside an Account but is not an entity on its own,
+  // a SubAccount (or TokenAccount) is living inside an Account but is not an entity on its own,
```

Comment:
```suggestion
  // a TokenAccount (or SubAccount, formerly) is living inside an Account but is not an entity on its own,
```

I think we should promote `TokenAccount` in this comment